### PR TITLE
82 feature pathway overview introduction component

### PR DESCRIPTION
**Pull request by Parham2002**

### CODE REVIEW

File: src/components/PathwayIntroduction/PathwayIntroduction.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import "./PathwayIntroduction.scss";
+import React from "react";
+
+const PathwayIntroduction = (props) => {
+  const { image, description } = props;
```
Comment (DorBia) [inline src/components/PathwayIntroduction/PathwayIntroduction.jsx:5]: Please destruct props in place - line above

File: src/components/PathwayIntroduction/PathwayIntroduction.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import "./PathwayIntroduction.scss";
+import React from "react";
```
Comment (DorBia) [inline src/components/PathwayIntroduction/PathwayIntroduction.jsx:2]: delete that line - not needed

File: src/components/PathwayIntroduction/PathwayIntroduction.test.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    const image = screen.getByAltText("pathway thumbnail");
+    expect(image).toBeInTheDocument();
+  });
+
+  it("renders the description", () => {
+    render(<PathwayIntroduction description={"Description"} />);
+    const description = screen.getByText("Description");
+    expect(description).toBeInTheDocument();
+  });
+});
```
Comment (DorBia) [inline src/components/PathwayIntroduction/PathwayIntroduction.test.jsx:22]: Can you add one test to see if the image is rendering? You can just check if queryByText("pathway thumbnail") is falsy

**Review by DorBia — CHANGES_REQUESTED** [review]
Just some small changes in comments above, and one more test needed

File: src/components/PathwayIntroduction/PathwayIntroduction.jsx (hunk 2 above)
Comment (garyinlam) [reply src/components/PathwayIntroduction/PathwayIntroduction.jsx]: removed unnecessary import

File: src/components/PathwayIntroduction/PathwayIntroduction.test.jsx (hunk 3 above)
Comment (garyinlam) [reply src/components/PathwayIntroduction/PathwayIntroduction.test.jsx:30]: added test to check if image renders correctly

**Review by nologyRob — APPROVED** [review]

**Review by DorBia — APPROVED** [review]

**Pull request merged**